src/Contacts/TSThread.m

```diff
@@ -19,8 +19,10 @@ @interface TSThread ()
 @property (nonatomic, retain) NSDate *creationDate;
 @property (nonatomic, copy) NSDate *archivalDate;
 @property (nonatomic, retain) NSDate *lastMessageDate;
-@property (nonatomic, copy) NSString *latestMessageId;
 @property (nonatomic, copy) NSString *messageDraft;
+
+- (TSInteraction *) lastInteraction;
+
 @end
```

Comment:
Why would you store another serialization of that interaction? I don't see why you would duplicate it in the database in the thread.